packages/jazz-tools/src/implementation/ContextManager.ts

```diff
@@ -44,6 +44,7 @@ export class JazzContextManager<
   protected props: P | undefined;
   protected authSecretStorage = new AuthSecretStorage();
   protected authenticating = false;
+  protected prevContextCreation?: Promise<void>;
```

Comment:
We can remove this.

Out of order context creation needs a more complete solution